docs/UserGuide.adoc

```diff
@@ -467,20 +467,25 @@ A graph is shown.
 // end::graph[]
 
 === Adding a reminder: `reminder add`
-Adds a reminder to the reminder list
+Adds a reminder to the reminder list when user enters a new reminder. Deadline and reminder message are recorded into storage.
+
 *Format:* `reminder add d/DATE m/MESSAGE`
 
+[NOTE]
+The date of a deadline must be within 5 years from the current date. MoneyGoWhere will check if the given reminder message and deadline are valid. If it encounters invalid information such as empty message, the application will notify with error.
```

Comment:
If it encounters invalid information such as **an** empty message, the application will **display an** error.